src/helpers/handleIncrementalUpdate.ts

```diff
@@ -23,7 +23,7 @@ export const handleIncrementalUpdate = (depthOld: string[][], newLevel: string[]
         result.splice(index, 1);
     }
 
-    return result;
+    return result.filter(order => order[1]); //order[0] - price, order[1] - amount
```

Comment:
No needs for this filtering here, you already have check for the second element at the start of the function.
Regarding filter - look at the 2-nd comment

Reply:
Yes, it is exist, but it is empty. Ex.: ["17", ""] => length will be 2, but we will receive zero amount value at the orderbook